health/micro-ui/web/micro-ui-internals/packages/modules/pgr/src/configs/UIcustomizations.js

```
@@ -0,0 +1,131 @@
+        case "WF_INBOX_HEADER_LOCALITY":
+          return value ? <span>{t(`${value}`)}</span> : <span>{t("NA")}</span>;
+
+        case "CS_COMPLAINT_DETAILS_CURRENT_STATUS":
+          return value && value?.length>0
+            ? <span>{t(`WF_INBOX_${value}`)}</span>: <span>{t("NA")}</span>;
+
+        case "WF_INBOX_HEADER_CURRENT_OWNER":
+          return value ? <span>{value?.[0]?.name}</span> : <span>{t("NA")}</span>;
+
+        case "WF_INBOX_HEADER_CREATED_DATE":
+          const dateLabel = Number.isFinite(value) && value > 0 ? new Date(value).toLocaleDateString() : t("ES_COMMON_NA");
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)